storage/internal/benchmarks/download_benchmark.go

```diff
@@ -18,39 +18,72 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"os"
 	"time"
 
 	"cloud.google.com/go/storage"
 )
 
-func downloadBenchmark(ctx context.Context, o *storage.ObjectHandle, objectSize int64) (elapsedTime time.Duration, err error) {
-	var bytesRead int64
-	var objectReader *storage.Reader
+type downloadOpts struct {
+	client     *storage.Client
+	objectSize int64
+	bucket     string
+	object     string
+}
+
+func downloadBenchmark(ctx context.Context, dopts downloadOpts) (elapsedTime time.Duration, rerr error) {
+	// Set timer
 	start := time.Now()
+	defer func() { elapsedTime = time.Since(start) }()
```

Comment:
Might be obvious but this could probably use a comment about the order in which the `defer` statements execute, because it's highly relevant here :)